downstream/modules/platform/ref-controller-dynamic-inventory.adoc

```diff
@@ -0,0 +1,19 @@
+[id="ref-controller-dynamic-inventory"]
+
+= Dynamic Inventory and private IP addresses
```

Comment:
This is identical to that of `30.12. Private EC2 VPC Instances in the controller Inventory` in the troubleshooting section.

Reply:
I'll comment it out of the assembly.